Command/InitializeCommand.php

```diff
@@ -20,6 +21,20 @@
  */
 class InitializeCommand extends ContainerAwareCommand
 {
+    private $rbacInitializer;
+
+    /**
+     * InitializeCommand constructor.
+     * @param null|string $name
+     * @param RbacInitializer $rbacInitializer
+     */
+    public function __construct(?string $name = null, RbacInitializer $rbacInitializer)
```

Comment:
why need constructor? If use constructor change ContainerAwareCommand to Command and register command to DI.